src/test/java/io/vertx/ext/mongo/MongoClientTest.java

```diff
@@ -77,6 +77,27 @@ public void testFindBatchResumePause() throws Exception {
     });
   }
 
+  @Test
+  public void findBatchResumePause() throws Exception {
+    int docs= 111;
+    testFindBatch((latch, stream) -> {
+      List<String> foos = new ArrayList<>();
```

Comment:
Is it really foos? more like test data or dummy data right? :-)

Reply:
make variable names great again